### ISSUE TITLE
v19.8.0 create-nx-workspace generate a non working environment - npm i in generated project crashes

### ISSUE THREAD

**Issue by thomas-lg** [nrwl/nx#28071]
### Current Behavior

When I run a `npm i` after I've generated my project, the command crash

### Expected Behavior

`npm i` should work

### GitHub Repo

[Stackblitz](https://stackblitz.com/edit/stackblitz-starters-sgwwje?file=angular-monorepo%2Fpackage.json)
`cd angular-monorepo`
`npm i`

### Steps to Reproduce

1. `npx create-nx-workspace@latest front --preset=angular-monorepo`
2. 
✔ Application name · aa-bb
✔ Which bundler would you like to use? · esbuild
✔ Default stylesheet format · scss
✔ Do you want to enable Server-Side Rendering (SSR) and Static Site Generation (SSG/Prerendering)? · No
✔ Test runner to use for end to end (E2E) tests · none
✔ Which CI provider would you like to use? · skip
✔ Would you like remote caching to make your build faster? · skip
3. `cd aa-bb`
4. `npm i`

### Nx Report

```shell
NX   Report complete - copy this into the issue template

Node           : 20.17.0
OS             : linux-x64
Native Target  : x86_64-linux
npm            : 10.8.2

nx                 : 19.8.0
@nx/js             : 19.8.0
@nx/jest           : 19.8.0
@nx/linter         : 19.8.0
@nx/eslint         : 19.8.0
@nx/workspace      : 19.8.0
@nx/angular        : 19.8.0
@nx/devkit         : 19.8.0
@nx/eslint-plugin  : 19.8.0
@nrwl/tao          : 19.8.0
@nx/web            : 19.8.0
@nx/webpack        : 19.8.0
typescript         : 5.5.4
```

### Failure Logs

```shell
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @typescript-eslint/utils@7.18.0
npm error Found: eslint@9.11.1
npm error node_modules/eslint
npm error   dev eslint@"^9.8.0" from the root project
npm error   peer eslint@"^8.57.0 || ^9.0.0" from @angular-eslint/builder@18.3.1
npm error   node_modules/@angular-eslint/builder
npm error     @angular-eslint/builder@"18.3.1" from angular-eslint@18.3.1
npm error     node_modules/angular-eslint
npm error       dev angular-eslint@"^18.3.0" from the root project
npm error   14 more (@angular-eslint/eslint-plugin, ...)
npm error
npm error Could not resolve dependency:
npm error peer eslint@"^8.56.0" from @typescript-eslint/utils@7.18.0
npm error node_modules/@typescript-eslint/utils
npm error   dev @typescript-eslint/utils@"^7.16.0" from the root project
npm error   peer @typescript-eslint/utils@"^7.11.0 || ^8.0.0" from @angular-eslint/eslint-plugin@18.3.1
npm error   node_modules/@angular-eslint/eslint-plugin
npm error     @angular-eslint/eslint-plugin@"18.3.1" from @angular-eslint/schematics@18.3.1
npm error     node_modules/@angular-eslint/schematics
npm error       @angular-eslint/schematics@"18.3.1" from angular-eslint@18.3.1
npm error       node_modules/angular-eslint
npm error     1 more (angular-eslint)
npm error   2 more (@angular-eslint/eslint-plugin-template, @angular-eslint/utils)
npm error
npm error Conflicting peer dependency: eslint@8.57.1
npm error node_modules/eslint
npm error   peer eslint@"^8.56.0" from @typescript-eslint/utils@7.18.0
npm error   node_modules/@typescript-eslint/utils
npm error     dev @typescript-eslint/utils@"^7.16.0" from the root project
npm error     peer @typescript-eslint/utils@"^7.11.0 || ^8.0.0" from @angular-eslint/eslint-plugin@18.3.1
npm error     node_modules/@angular-eslint/eslint-plugin
npm error       @angular-eslint/eslint-plugin@"18.3.1" from @angular-eslint/schematics@18.3.1
npm error       node_modules/@angular-eslint/schematics
npm error         @angular-eslint/schematics@"18.3.1" from angular-eslint@18.3.1
npm error         node_modules/angular-eslint
npm error       1 more (angular-eslint)
npm error     2 more (@angular-eslint/eslint-plugin-template, @angular-eslint/utils)
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /home/tlegougaud/.npm/_logs/2024-09-24T09_48_54_253Z-eresolve-report.txt
npm error A complete log of this run can be found in: /home/tlegougaud/.npm/_logs/2024-09-24T09_48_54_253Z-debug-0.log
```

### Operating System

- [ ] macOS
- [X] Linux
- [ ] Windows
- [ ] Other (Please specify)

### Additional Information
Removing `@typescript-eslint/utils` from the devDependencies fix the issue

**Comment by jaysoo** [comment]
Hi @thomas-lg , it looks like the version we're installing for `@typescript-eslint/utils` is outdated. If you update the version to `^8.0.0` then the error should go away.

**Comment by thomas-lg** [comment]
Hi @jaysoo ,
Thanks for the tips! I'm just wondering if it is normal that the wrong version is embedded in the `package.json` generated by `create-nx-workspace`. I guess other might encounter the same issue 🤔.

**Comment by jaysoo** [comment]
No, we need to patch it on our end. When we updated to ESLint v9, the utils package seem to be missed.

**Comment by thomas-lg** [comment]
Alright then. Thanks for answering!
Love NX by the way 😄!

**Comment by jaysoo** [comment]
Glad to help, a patch is incoming.

**Comment by jaysoo** [comment]
This has been fixed here https://github.com/nrwl/nx/pull/28267

**Comment by jaysoo** [comment]
Released in latest 20.0.2 version.
